packages/ui-extensions/src/surfaces/checkout/components/QRCode/QRCode.doc.ts

```diff
@@ -41,11 +41,7 @@ const data: ReferenceEntityTemplateSchema = {
       type: 'Generic',
       anchorLink: 'best-practices',
       title: 'Best Practices',
-      sectionContent: `
-        - Always test that the QR code is scannable from a smartphone.
-        - Include a square logo if that’s what your customers are familiar with.
-        - Increase usability by adding a text description of what the QR code does and provide alternate methods to get to the same destination, like a text input containing the value.
-      `,
```

Comment:
I still think it does, I use it on [Switch](https://shopify.dev/docs/api/checkout-ui-extensions/unstable/components/forms/switch), can we check the formatting and try again? like [this](https://github.com/Shopify/ui-extensions/pull/2014/files#diff-c068355d6a213f4ba727caca947dc925244c28050b713840f5b365cfb70df0a7R45-R62). 

We can pair if you want

Reply:
Oh, you're right. The white space was the culprit. Updated so that it's multiline now without extra whitespacing.